src/main/kotlin/racingcar/domain/GameResult.kt

```diff
@@ -0,0 +1,48 @@
+package racingcar.domain
+
+class GameResult(
+    val records: List<List<Record>>
+) {
+    val winners = getWinnerFrom(records.last())
+
+    private fun getWinnerFrom(gameResult: List<Record>): List<String> {
+        val organizePosition = organizePosition(gameResult)
+        val highestPosition = organizePosition.first
+        val recordByPosition = organizePosition.second
```

Comment:
`Pair` 타입은 구조 분해로 다음처럼 대입할 수도 있습니다.

```suggestion
        val (highestPosition, recordByPosition) = organizePosition(gameResult)
```

Reply:
오 코드가 매우 간결해지네요 !!

Reply:
해당 [커밋](https://github.com/next-step/kotlin-racingcar/pull/632/commits/7f90bddb3780d92faf2fa8804b3f29872c80474a)에 반영하였습니다.